src/components/Chip/Chip.tsx

```diff
@@ -1,37 +1,71 @@
 import { ChipProps } from './types';
 import styled from 'styled-components';
+import React from 'react';
+import * as cssVariables from '@/static/styles/variables';
+import DeleteActiveIconImg from '@/static/img/icon-delete-active.svg';
 
 /**
  * Layout to one Skill Chip
  */
 
-// eslint-disable-next-line react/prop-types
 const Chip: React.FC<ChipProps> = ({ text }) => {
-  return (
-    <StyledChip key={text} value={text}>
-      {text}
-    </StyledChip>
-  );
+  const [chipSelected, setChipSelected] = React.useState(false);
+
+  const handleChipClick = () => setChipSelected(!chipSelected);
+
+  if (chipSelected) {
+    return (
+      <StyledSelectedChip key={text} value={text} onClick={handleChipClick}>
+        {text}
+      </StyledSelectedChip>
+    );
+  } else {
+    return (
+      <StyledChip key={text} value={text} onClick={handleChipClick}>
+        {text}
+      </StyledChip>
+    );
+  }
```

Comment:
Instead of having two different styled-components, you can also add props to the styled components like this:
<img width="556" alt="Screenshot 2022-07-28 at 14 09 37" src="https://user-images.githubusercontent.com/34128180/181491636-c2f645e2-5271-48a0-bdc4-66db0cb5a91c.png">

Then you can change the style depending on the props -)

Additionally I always like the booleans to be named so that they start with _is_, _has_, _should_ etc.

Reply:
I did not browse the design, but now the palepurple is a bit pale and the font does not seem to work with it. I am pretty sure it is not accessible.